# Copyright (c) 2017 Intel Corporation
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

SonarFilters:
   - {subscribe: "/water_uavcan_master/ultrasonic_l1", publish: "/water_uavcan_master/sonar_filtered_l1"}
   - {subscribe: "/water_uavcan_master/ultrasonic_m",  publish: "/water_uavcan_master/sonar_filtered_m"}
   - {subscribe: "/water_uavcan_master/ultrasonic_r1", publish: "/water_uavcan_master/sonar_filtered_r1"}

base_frame: /base_link
ultrasonic_l1_frame: ultrasonic_l1
ultrasonic_m_frame:  ultrasonic_m
ultrasonic_r1_frame: ultrasonic_r1

